tests/python-pytest/onnx/import/test_cases.py

```diff
@@ -81,26 +79,27 @@
     'test_cast',
     'test_instancenorm',
     #pytorch operator tests
-    'test_operator_exp',
+    'test_exp',
     'test_operator_maxpool',
     'test_operator_params',
     'test_operator_permute2',
     'test_depthtospace',
-    'test_size'
+    'test_size',
+    'test_log',
+    'test_gather',
+    'test_dropout',
+    'test_leakyrelu',
+    'test_elu'
     ]
 
 BASIC_MODEL_TESTS = [
     'test_AvgPool2D',
     'test_BatchNorm',
     'test_ConstantPad2d'
     'test_Conv2d',
-    'test_ELU',
```

Comment:
why remove these tests?

Reply:
there doesn't seem to be a test_Elu in the standard onnx test list. And test_Elu is not calling the import function _elu(). changed this test to test_elu in implemented_operators_test list

Reply:
it is a model test not an operator test. can the model tests please be reverified?

Reply:
test_elu now has an onnx operator test. therefore, removing it from basic model test.
As discussed, i will add the list of tests added/deleted in the PR description